README.md

```diff
@@ -481,7 +481,7 @@ page such as what might be generated serverside by platforms like Drupal,
 WordPress etc.
 
 For this use case you should download the `dist.zip` package from
-[the latest release of the project](/danskernesdigitalebibliotek/ddb-react/releases/latest)
+[the latest release of the project](/danskernesdigitalebibliotek/dpl-react/releases/latest)
 and unzip somewhere within the web root of your project. The package contains a
 set of artifacts needed to use one or more applications within an HTML page.
 
```

Comment:
How about these babies? 
![image](https://user-images.githubusercontent.com/28546954/165763821-7465ac6f-0c42-4ae4-9b87-1486b0870dad.png)

Reply:
"I managed to find some instances where we still use ddb instead of dpl - is that deliberate?"
Good catch. I have fixed it.

Reply:
_There are also 3 tests that failed. In at least one of these I found it had something to do with not being able to find things in folders?_
I'll deactivate the Cypress steps until we have our first tests.